packages/marble-api/src/generated/marble-api.ts

```diff
@@ -109,46 +109,55 @@ export type CreateCaseBody = {
 export type CaseEventDtoBase = {
     id: string;
     case_id: string;
-    user_id: string;
     created_at: string;
     event_type: string;
 };
 export type CaseCreatedEvent = {
     event_type: "case_created";
-} & CaseEventDtoBase;
+} & CaseEventDtoBase & {
+    user_id: string;
+};
```

Comment:
After reading the backend PR, I realized you deleted specific events for "wrokflow" and merged them inside existing ones.
So I think `user_id` should be optionnal here

Reply:
ok my bad, it's up to date in the openapi file but I must have missed a round of api generation